content/participate_member.php

```diff
@@ -129,15 +125,31 @@
         </select>
     </div>
     <input type="submit" class="submit" value="Herunterladen">
-    <p>Eine Anleitung wie du die Freifunk Flensburg Firmware auf deinem Router selber installieren kannst findest du bei uns <a href="http://wiki.freifunk-flensburg.de/wiki/Einen_Knoten_einrichten">hier im Wiki</a>. Die Freifunk Flensburg Firmware steht unter der <a href="http://de.wikipedia.org/wiki/GNU_General_Public_License" target="_blank">GPL Lizenz</a>. Du darfst sie somit kostenlos nutzen, studieren, ändern und verbreiten.</p>
+    <p>Eine Anleitung wie du die <?=$site_title?> Firmware auf deinem Router selber installieren kannst findest du bei uns <a href="<?=$config['wiki']['url']?>Einen_Knoten_einrichten">hier im Wiki</a>. Die <?=$site_title?> Firmware steht unter der <a href="http://de.wikipedia.org/wiki/GNU_General_Public_License" target="_blank">GPL Lizenz</a>. Du darfst sie somit kostenlos nutzen, studieren, ändern und verbreiten.</p>
 </form>
 <aside class="participate_list-links">
-    <h2>Links zu Freifunk Flensburg</h2>
+    <h2>Links zu <?=$site_title?></h2>
     <ul>
-        <li><a target="_blank" href="http://wiki.freifunk-flensburg.de/wiki/Hauptseite">Freifunk Flensburg Wiki</a> - Unser gesammeltes Wissen</li>
-        <li><a target="_blank" href="https://github.com/freifunk-flensburg">GitHub</a> - Unser Code zum Mitbasteln</li>
-        <li><a target="_blank" href="http://map.freifunk-flensburg.de">Die Mesh-Map</a> - Unsere detailierte Knotenkarte</li>
-        <li><a target="_blank" href="http://nordlab-ev.de">nordlab e. V.</a></li>
+        <li>
+            <a target="_blank" href="<?=$config['wiki']['url']?>Hauptseite"><?=$config['wiki']['content']?></a><?
```

Comment:
Halte es für unnötig hier explizit "Hauptseite" der URL anzufügen, da das Wiki ja nicht MediaWiki und dann auch noch in deutsch sein muss.